common/txmgr/types/forwarder_manager.go

```diff
@@ -0,0 +1,13 @@
+package types
+
+import "context"
+
+type ForwarderManager[ADDR any] interface {
+	Name() string
```

Comment:
These Name, Start, Stop can be replaced by services.ServiceCtx

Reply:
+1 to this